ts/components/ui/Markdown/index.tsx

```diff
@@ -31,13 +31,25 @@ const INJECTED_JAVASCRIPT = `
 ${NOTIFY_INTERNAL_LINK_CLICK_SCRIPT}
 `;
 
+const TITILLIUM_WEB_FONT_PATH =
+  Platform.OS === "android"
+    ? "file:///android_asset/fonts/TitilliumWeb-Regular.ttf"
+    : `${RNFS.MainBundlePath}/TitilliumWeb-Regular.ttf`;
```

Comment:
did you checked that this still work in release mode without the filesystem access flag?

Reply:
Yes i tested the io-icon-font and it is working on the beta-release version.